scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -0,0 +1,172 @@
+            # set the base for special cases like median, etc.
+            if not measurement_assigned and 'measurement' in self.features:
+                if part in self.features['measurement']:
+                    stat_var.update(self.features['measurement'][part])
+                    measurement_assigned = True
+
+        # set the default statVar definition
+        if not measurement_assigned and 'measurement' in self.features and '_DEFAULT' in self.features[
+                'measurement']:
+            stat_var.update(self.features['measurement']['_DEFAULT'])
+
+        # set the populationType attribute
+        if 'populationType' not in stat_var:
+            stat_var['populationType'] = self._get_population_type(part_list)
+
+        # associate pvs to stat_var
+        if 'pvs' in self.features:
+            #p = property, k = dictionary of substrings in column that points to a propertyValue
+            for p, k in self.features['pvs'].items():
+                for c, v in k.items():
+                    ## check if the current key of dict matches with any substring from the tokens of a column name
+                    for part in part_list:
+                        # check if column substring matches with a column token
+                        # the check is done by ignoring the case
+                        if c.lower() in part.lower():
+                            stat_var[p] = v
+                            # add inferred properties from property 'p' to the stat_var
+                            if 'inferredSpec' in self.features and p in self.features[
+                                    'inferredSpec']:
+                                stat_var.update(
+                                    self.features['inferredSpec'][p])
+
+        #TODO: If quantityRanges occurs as values in multiple properties, find the property that relates to the column
+        #Suggestion: using _DISAMBIGUATING_POP_TYPE for pvs (income and earnings -- in the test)
+
+        #TODO: Add units and scalingFactor
+
+        # if the column is Margin of Error, update statType key
+        stat_var = self._update_stat_var_dict(stat_var, part_list)
+        ## add Universe PVs based on the populationType of StatVar
+        dependent_properties = None
+        if 'universePVs' in self.features:
+            for elem in self.features['universePVs']:
+                if stat_var['populationType'] == elem['populationType']:
+                    # check if all constraints of this populationType is in stat_var
+                    if (set(elem['constraintProperties']).issubset(
+                            set(list(stat_var.keys())))):
+                        #TODO: add dependent PVs to the stat_var dict if it is not present
+                        dependent_properties = list(elem['dependentPVs'].keys())
+
+        # generating dcid using the utils/statvar_dcid_generator.py
+        stat_var['dcid'] = get_stat_var_dcid(stat_var,
```

Comment:
The property for dcid is 'Node'
stat_var['Node'] = 'dcid:' +  get_stat_var_dcid(...)